src/coreclr/gc/vxsort/do_vxsort_neon.cpp

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if defined(TARGET_ARM64)
```

Comment:
Maybe just conditionally compile them (in cmake) instead of adding `ifdef`s?

Reply:
> Maybe just conditionally compile them (condition in cmake) instead of adding `ifdef`s?

I'm used to seeing files elsewhere in coreclr be compiled for all targets and just have an ifdef at the top. Eg, anything in jit/ with a cpu in the filename. So, was following that convention. But happy to switch.